src/frontmatter.xml

```diff
@@ -35,84 +35,87 @@
--->
-        <!-- Removed reference to the solutions manual.  TWJ 8/19/2010 -->
-        <!-- A complete solutions manual is available for the instructor's use.  -->
-<!--
-        <p>There are additional exercises or computer projects at the ends of many of the chapters. The computer projects usually require a knowledge of programming. All of these exercises and projects are more substantial in nature and allow the exploration of new results and theory.</p>
--->
-        <!-- % Added Sage blurb, RAB 2011/07/28 -->
-<!--        <p>Sage (<url href="http://sagemath.org">sagemath.org</url>) is a free, open source, software system for advanced mathematics, which is ideal for assisting with a study of abstract algebra. Sage can be used either on your own computer, a local server, or on SageMathCloud (<url href="https://cloud.sagemath.com">https://cloud.sagemath.com</url>). Robert Beezer has written a comprehensive introduction to Sage and a selection of relevant exercises that appear at the end of each chapter, including live Sage cells in the web version of the book.</p>
--->
-        <!-- RAB 2014/08/18 Should move flush right with some vertical spacing -->
-<!--        <attribution>Thomas W. Judson <br />Nacogdoches, Texas 2015</attribution>
-
-    </preface>-->
+    <acknowledgement>
+        <p>Over several terms, Jack Green wrote most of the book's content as a Word/PDF document for use in his face-to-face and online Precalculus courses at Mt. Hood Community College.</p>
+        <p>In 2016/2017, Nick Chura, Jack Green and Dr. Alex Jordan converted the material to MathBook XML (enabling the HTML output alongside a PDF) with support from an Open Oregon grant.</p>
+        <p>Special thanks to Dr. Rob Beezer, David Favreault, Jim Hatton, Robert Hauss, Dr. Alex Jordan, Dr. Elise Lockwood and Gina Shankland for proofreading and review.</p>
+    </acknowledgement>
+
+    <preface xml:id="to-all">
+        <title>To All</title>
+        <p>Precalculus is taught at Mt. Hood Community College over two <m>10</m><ndash />week courses. This text aims to address all learning outcomes from the first course, suitable for use in a single <m>10</m> week term.</p>
+
+        <paragraphs>
+            <title>HTML and PDF</title>
+            <p>This text will be released with several synchronous versions that offer different features. The essential content of each version is the same as for all others.</p>
+            <ul>
+                <li><p>Currently, a preview of the web version is available at <url href="http://www.greenprecalc.com/Math%20111/precalc1-MHCC/html/"/>. The web version offers full walk-through solutions to most exercises, as well as interactive elements and easier navigation than print versions.</p></li>
```

Comment:
I'm going to move the url. Let's not mention "111" since that may be Oregon-specific. I think just `http://www.greenprecalc.com/precalc1-MHCC/html/` would be good. If you change this to that, then that is how I will move things on the site.